components/org.wso2.identity.event.common.publisher/src/main/java/org/wso2/identity/event/common/publisher/EventPublisherService.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.identity.event.common.publisher;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.identity.event.common.publisher.internal.EventPublisherDataHolder;
+import org.wso2.identity.event.common.publisher.model.EventContext;
+import org.wso2.identity.event.common.publisher.model.SecurityEventTokenPayload;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+/**
+ * Event Publisher Service.
+ */
+public class EventPublisherService {
+
+    private static final Log log = LogFactory.getLog(EventPublisherService.class);
+    private static final int THREAD_POOL_SIZE = 10; // Customize the thread pool size based on your needs
```

Comment:
IMO, we don't need to have a comment like "Customize the thread pool size based on your needs"

Reply:
Addressed with https://github.com/wso2-extensions/identity-event-publishers/pull/1/commits/7999427bc5211016a93107fb27f9efd67513d525